### DISCUSSION TITLE
1.1.0

### DISCUSSION THREAD

**Discussion by lonqie** [lonqie/SchaleDB#87] [Updates]
## New Features & Improvements

- Added a new setting to redirect to the last visited page when opening Schale DB.
- Added a new setting to choose whether to display the list of items or last viewed item automatically when navigating to a module.
- Added a new setting to limit the display width of page content. (This will mimic the layout of the previous version of Schale DB)
- Light theme colours have been adjusted slightly to be less bright and improve contrast in some areas.

### Students
- Added an experimental setting to show application delay (the time between a skill being activated and the effect being applied) for buff and debuff skills. **This must be switched on in the settings.**
- "Can Heal" and "Can Dispel" filters can now have the target specified.
- "Can Reposition" and "Has Summon" filters have been added.
- "No. of Alts" sort option has been added.
- "Released" sort order has been renamed and made default (as on schale.gg), "Default" order has been renamed "ID".
- Added normal attack ammo and timing data to the character skills tab.

### Items
- Added display of equipment tiers to random item tooltips.
- Added a location level slider to schedule area drop rates.

### Damage Calculator
- Added total damage details table to calculation results that displays the minimum, maximum and 0%/100% crit averages
- Added display of EX Skill cost to calculation results and tooltips.
- Changed behaviour of User Defined buffs form to stay open after adding.

## Fixes
- Fixed incorrect sorting order for buff and debuff based filters.
- Fixed an issue with the server reset clock.
- Fixed several layout display issues on certain screen breakpoints.

<hr /><em>This discussion was created from the release <a href='https://github.com/SchaleDB/SchaleDB/releases/tag/1.1.0'>1.1.0</a>.</em>

**Comment by Gongqiannian** [comment]
nice
